packages/runtime/container-runtime/src/orderedClientElection.ts

```diff
@@ -331,17 +331,7 @@ export class OrderedClientElection
         if (typeof initialState === "number") {
             this._electionSequenceNumber = initialState;
         } else {
-            // Override the initially elected client with the initial state.
-            if (initialClient?.clientId !== initialState.electedClientId) {
-                // Cannot find initially elected client, so elect undefined.
-                logger.sendErrorEvent({
-                    eventName: "InitialElectedClientNotFound",
-                    electionSequenceNumber: initialState.electionSequenceNumber,
-                    expectedClientId: initialState.electedClientId,
-                    electedClientId: initialClient?.clientId,
-                    clientCount: orderedClientCollection.count,
-                });
-            } else if (initialClient !== undefined && !isEligibleFn(initialClient)) {
+            if (initialClient !== undefined && !isEligibleFn(initialClient)) {
```

Comment:
I do not know well this code. Is this change intentional? Can you please share more on why it's made?

Reply:
Snapshot is referring to elected summarizer ID that is not present/valid. To help wrap up this PR I just removed this change. I'll bring in runtime changes as a separate PR.